resources/lang/en/filament-value-range-filter.php

```diff
@@ -3,8 +3,18 @@
 return [
 
     'range.options.equal' => 'is equal to',
+    'range.options.not_equal' => 'is not equal to',
     'range.options.between' => 'is between',
     'range.options.greater_than' => 'is greater than',
+    'range.options.greater_than_eqal' => 'is greater than or equal to',
     'range.options.less_than' => 'is less than',
+    'range.options.less_than_equal' => 'is less than or equal to',
+    'range.indicator.equal' => ':label is equal to :value',
+    'range.indicator.not_equal' => ':label is not equal to :value',
+    'range.indicator.between' => ':label is between :formValue and :toValue',
```

Comment:
Thank you! :)

Could you fix the typo `:formValue` to `:fromValue` in
`':label is between :formValue and :toValue',`

Reply:
Sure :)

Reply:
Thank you :)